src/bridge/bridge.c

```diff
@@ -557,11 +557,13 @@ run_bridge (const gchar *interactive,
       transport = cockpit_pipe_transport_new_fds ("stdio", 0, outfd);
     }
 
+#if WITH_POLKIT
```

Comment:
It's fine to move the `gpointer polkit_agent = NULL;` here, so that it stays within the `#if`. Modern C allows that, and it's cleaner in this case IMHO.

Reply:
You are right.